challenge_help_me_carry/src/challenge_hmc_functions/hmc_functions.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
+
+import smach
+import datetime
+import robot_smach_states as states
+import robot_smach_states.util.designators as ds
+
+from robot_smach_states.util.designators import check_type
+from robot_skills.arms import Arm, GripperState
+from hmi import TimeoutException
+from robocup_knowledge import load_knowledge
+from robot_skills.util import kdl_conversions
+from robot_skills.util.entity import Entity
+
+challenge_knowledge = load_knowledge('challenge_help_me_carry')
```

Comment:
After apply all other commnets, this can be removed from this file. It should only be in the main file.